r_markdown/load_save_data.Rmd

```diff
@@ -211,6 +211,43 @@ Reference for test id, stove and fuel types.
                              "Liquefied Petroleum Gas")))
 ```
 
+* Define stove fuel combination category levels
+
+```{r, echo=TRUE, warning=FALSE, message=FALSE, results='markup', cache=FALSE}
+  samples <- dplyr::mutate(samples, stove_fuel = as.factor(paste(stove, ":", fuel))) %>%
+             dplyr::mutate(stove_fuel = factor(stove_fuel,
+                           levels = c("Three Stone Fire (Artisan) : Douglas Fir",
+                             "Three Stone Fire (Artisan) : Oak",
+                             "Three Stone Fire (Artisan) : Eucalyptus",
+                             "Clay Ring (Artisan) : Douglas Fir",
+                             "Clay Ring (Artisan) : Oak",
+                             "Clay Ring (Artisan) : Eucalyptus",
+                             "Built-in Justa (Envirofit, HM5000) : Douglas Fir",
+                             "Built-in Justa (Envirofit, HM5000) : Oak",
+                             "Built-in Justa (Envirofit, HM5000) : Eucalyptus",
+                             "Rocket Elbow (Envirofit, G3300) : Douglas Fir",
+                             "Rocket Elbow (Envirofit, G3300) : Oak",
+                             "Rocket Elbow (Envirofit, G3300) : Eucalyptus",
```

Comment:
What's going on here? Could you have done the formatting in the paste statement you used to create the variable?